modules/developer-cli-odo-listing-available-services-from-the-operators-installed-on-the-cluster.adoc

```diff
@@ -17,7 +17,7 @@ $ odo catalog list services
 The command lists Operators and the CRDs.
 The output of the command shows the Operators installed on your cluster. For example:
 +
-----             
+----
```

Comment:
Did you miss a file? The output and command need the new mark up.

Reply:
I think this got sucked in cuz I accidentally touched the file while looking at it. Removing it from this PR since it's the CLI Ref guide.